### DISCUSSION TITLE
`getsentry/paths-filter` seems to be unmaintained

### DISCUSSION THREAD

**Discussion by kezhenxu94** [apache/skywalking#10604] [Ideas]
`getsentry/paths-filter` seems to be unmaintained and there are many warnings from this GHA, some of the warnings include deprecated usage of command `set-output` that might be removed in the future. We should come out with another GHA to filter the paths. /cc @Superskyyy

**Comment by Superskyyy** [comment]
I searched for alternatives but there isn't really any compatible solutions to path filtering :( Since few big names also use this paths-filter, we can maybe wait for their action and see what is the best. (Chrome web.dev and Sentry).

**Reply by Superskyyy** [reply]
Otherwise we can try fork it to apply the pending PRs in the project.
